core/storage/suggestion/gae_models_test.py

```diff
@@ -6169,3 +6169,39 @@ def test_get_field_name_mapping_to_takeout_keys(self) -> None:
             {
                 'coordinator_ids': 'coordinated_language_ids'
             })
+
+    def test_get_returns_model_when_it_exists(self) -> None:
+
+        translation_coordinators_model = (
+            suggestion_models.TranslationCoordinatorsModel.get(
+                self.LANGUAGE_4_ID
+            )
+        )
+
+        # Ruling out the possibility of None for mypy type checking.
+        assert translation_coordinators_model is not None
+        self.assertEqual(
+            translation_coordinators_model.id,
+            self.LANGUAGE_4_ID
+        )
+        self.assertEqual(
+            translation_coordinators_model.coordinator_ids,
+            [self.USER_ID_2]
+        )
+        self.assertEqual(
+            translation_coordinators_model.coordinators_count,
+            1
+        )
```

Comment:
I think these 2 assert statements don't need to be there. It's enough to check that we got the same exact model by matching the model IDs. By removing the 2 assert statements, the test will make it abundantly clear to the reader that each `TranslationCoordinatorModel` is identified by its language.

Reply:
Sg. Thanks.